### ISSUE TITLE
Feature request: Image folding support (fold data: URL)

### ISSUE THREAD

**Issue by jorgk3** [ajaxorg/ace#4755]

**Comment by InspiredGuy** [comment]
The links in the issue lead to 404 page, could you please provide more context on the requested feature itself?

**Comment by jorgk3** [comment]
Yes, "native" image folding would be lovely to have. Currently we're hacking it in ThunderHTMLedit.

**Comment by whazor** [comment]
To clarify, this would be code folding support in the HTML and CSS modes for:
```
<img src="data:image/jpeg;base64,/9j/4QCWRXhpZgAATU0AKgAAAAgABgEOAAIAAAAYAAAAVgEaAAUAAAABAAAAbgEbAAUAAAABAAAAdgEoAAMAAAABAAI ..."/>
```

Which would be useful as those images can be very big (800 KB image would be a lot of base64 letters).

That sounds like a useful feature yes.

**Comment by jorgk3** [comment]
We hack it into place and it looks like this:
![image](https://user-images.githubusercontent.com/8466558/222928861-021ddaab-fe02-4527-a532-191b9e41fe27.png)
![image](https://user-images.githubusercontent.com/8466558/222928896-0bc76b32-0015-42f4-bd09-7572c6c84d91.png)

Please refer to https://github.com/ajaxorg/ace-builds/issues/147#issuecomment-448415677 and below.

**Comment by Betterbird** [comment]
> That sounds like a useful feature yes.

👍

**Comment by jorgk3** [comment]
> That sounds like a useful feature yes.

👍
